From: Otto Kekalainen <otto@debian.org>
Date: Wed, 2 Mar 2022 18:13:18 -0800
Subject: Fix various spelling errors still found in code

Two upstream PRs remain that have been merged, but not imported on MariaDB yet.

Forwarded: https://github.com/facebook/rocksdb/pull/9653
Origin: https://patch-diff.githubusercontent.com/raw/facebook/rocksdb/pull/9653.patch
# Merged in RocksDB 7.3.1, but not updated into MariaDB yet
---
 storage/rocksdb/rocksdb/db/external_sst_file_ingestion_job.cc | 4 ++--
 storage/rocksdb/rocksdb/include/rocksdb/cache.h               | 2 +-
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/storage/rocksdb/rocksdb/db/external_sst_file_ingestion_job.cc b/storage/rocksdb/rocksdb/db/external_sst_file_ingestion_job.cc
index 4cec5d3..86aec50 100644
--- a/storage/rocksdb/rocksdb/db/external_sst_file_ingestion_job.cc
+++ b/storage/rocksdb/rocksdb/db/external_sst_file_ingestion_job.cc
@@ -46,7 +46,7 @@ Status ExternalSstFileIngestionJob::Prepare(
             TablePropertiesCollectorFactory::Context::kUnknownColumnFamily &&
         f.cf_id != cfd_->GetID()) {
       return Status::InvalidArgument(
-          "External file column family id dont match");
+          "External file column family id don't match");
     }
   }
 
@@ -646,7 +646,7 @@ Status ExternalSstFileIngestionJob::AssignGlobalSeqnoForIngestedFile(
     return Status::InvalidArgument("Global seqno is required, but disabled");
   } else if (file_to_ingest->global_seqno_offset == 0) {
     return Status::InvalidArgument(
-        "Trying to set global seqno for a file that dont have a global seqno "
+        "Trying to set global seqno for a file that don't have a global seqno "
         "field");
   }
 
diff --git a/storage/rocksdb/rocksdb/include/rocksdb/cache.h b/storage/rocksdb/rocksdb/include/rocksdb/cache.h
index 77ddf52..ee27538 100644
--- a/storage/rocksdb/rocksdb/include/rocksdb/cache.h
+++ b/storage/rocksdb/rocksdb/include/rocksdb/cache.h
@@ -60,7 +60,7 @@ struct LRUCacheOptions {
   // If greater than zero, the LRU list will be split into a high-pri
   // list and a low-pri list. High-pri entries will be insert to the
   // tail of high-pri list, while low-pri entries will be first inserted to
-  // the low-pri list (the midpoint). This is refered to as
+  // the low-pri list (the midpoint). This is referred to as
   // midpoint insertion strategy to make entries never get hit in cache
   // age out faster.
   //
